packages/test/functional-tests/src/test/ddsHandleEncoding.spec.ts

```diff
@@ -102,25 +110,72 @@ describe("DDS Handle Encoding", () => {
 			},
 			[handle.absolutePath] /* expectedHandles */,
 		),
+		createTestCase(
+			new SharedStringFactory(),
+			(dds) => {
+				dds.insertMarker(0, ReferenceType.Simple, { marker: handle });
+			},
+			[handle.absolutePath] /* expectedHandles */,
+		),
+		createTestCase(
+			new SharedMatrixFactory(),
+			(dds: SharedMatrix) => {
+				dds.insertRows(0, 1);
+				dds.insertCols(0, 1);
+
+				dds.setCell(0, 0, handle);
+			},
+			[handle.absolutePath] /* expectedHandles */,
+		),
+		createTestCase(
+			new TreeFactory({}),
+			(dds: ITree) => {
+				const builder = new SchemaFactory("test");
+				class Bar extends builder.object("bar", {
+					h: builder.optional(builder.handle),
+				}) {}
+
+				const config = new TreeConfiguration(Bar, () => ({
+					h: undefined,
+				}));
+
+				const treeView = dds.schematize(config);
+
+				treeView.root.h = handle;
+			},
+			[handle.absolutePath] /* expectedHandles */,
+		),
 		createTestCase(
 			new ConsensusRegisterCollectionFactory(),
 			(dds) => {
-				dds.write("whatever", handle).catch(() => {
-					// We only care about errors before message submission, which will fail the message.length assert below.
-				});
+				dds.write("whatever", handle).catch(() => {});
+			},
+			[handle.absolutePath] /* expectedHandles */,
+		),
+		createTestCase(
+			new ConsensusQueueFactory(),
+			(dds) => {
+				dds.add(handle).catch(() => {});
 			},
+			// todo: AB#7149 this DDS does not currently support storing handles
```

Comment:
Personal preference: I've seen too many 5-year-old `// todo:` comments, I don't think they're an effective way to keep track of work.  Better just to say plainly "This DDS does not...".

If someone's curious about context, they can do git blame and see this PR.  Hopefully your description will mention something about this and other interesting cases.

Also I don't think we want AB links in the source code.